aten/src/ATen/native/quantized/cpu/qrelu.cpp

```diff
@@ -24,7 +24,7 @@ Tensor qnnpack_relu(Tensor input) {
   TORCH_CHECK(
       input.ndimension() > 0, "qnnpack_relu(): Got empty input tensor");
 
-  Tensor input_contig = input.contiguous();
+  Tensor input_contig = input.contiguous(input.suggest_memory_format());
```

Comment:
@kimishpatel are you saying this line breaks tests?

Reply:
Yes. I had to back out the change because it was breaking this test: `quantization.test_quantize.TestModelNumerics. test_float_quant_compare_per_tensor`. If I removed the second relu in the module (https://github.com/pytorch/pytorch/blob/87b198d309e00f52e098a533c22b087b36fc00e3/torch/testing/_internal/common_quantization.py#L773) for this test it worked. First one gets fused.

Reply:
I didn't have time to finish the investigation yet, but hope this helps:
* adding some debug code to narrow down the problem: https://gist.github.com/vkuzo/9e6fe1e4ef0a4754aef3026e97244240 .
* with the code above
`
# correct layer output
[0.8699, 0.8249, 0.8849, 0.8399, 0.7949, 0.7349, 0.7349, 0.7199, 0.8399, 0.8849, 0.7199, 0.7199]
# incorrect layer output
[0.8699, 0.7949, 0.8399, 0.8249, 0.7349, 0.8849, 0.8849, 0.7349, 0.7199, 0.8399, 0.7199, 0.7199]
# we see that incorrect layer output is the NHWC version of the correct NCHW layer output
`
so, something seems funky with the memory format of the output